mod/state-transition/pkg/core/state_processor.go

```diff
@@ -437,82 +432,3 @@ func (sp *StateProcessor[
 	)
 	return st.SetLatestBlockHeader(lbh)
 }
-
-// getAttestationDeltas as defined in the Ethereum 2.0 specification.
-// https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#get_attestation_deltas
-//
-//nolint:lll
-func (sp *StateProcessor[
-	_, _, _, BeaconStateT, _, _, _, _, _, _, _, _, _, _, _, _, _,
-]) getAttestationDeltas(
```

Comment:
Still used by the `examples/berad` module.

Provided by `examples/berad/pkg/state-transition/state_processor.go` row 458.

Called in `examples/berad/pkg/state-transition/state_processor.go` row 488.